subprojects/common/report-viewer/src/main/kotlin/com/avito/report/internal/DirectTestArtifactsProvider.kt

```diff
@@ -0,0 +1,40 @@
+package com.avito.report.internal
+
+import com.avito.android.Result
+import com.avito.report.TestArtifactsProvider
+import java.io.File
+import java.util.UUID
+
+class DirectTestArtifactsProvider(
```

Comment:
Can it be internal as hidden by TestArtifactsProviderFactory?

Reply:
yes